subsystem/manifests_test.go

```diff
@@ -138,6 +139,44 @@ spec:
 		})
 	})
 
+	It("cluster doesn't exist", func() {
+		non_exiting_id := strfmt.UUID(uuid.New().String())
+		By("create manifest returns not found")
```

Comment:
for clarity, maybe better to put each of these in a separate By func. E.g as done in https://github.com/openshift/assisted-service/pull/3510/files#diff-6173872f98a7065d377afcb41777af74922ed6d692ac80254da515c6d3477d88L153

Reply:
It has the same effect

Reply:
Yes, but the test output is more clear when they are separate. Also then you'll know if one or multiple are failing.
Currently if any expectation here fails then the rest of the tests don't get run.

Reply:
> Please separate out the test cases into their own `It` blocks for separate endpoints.
Each steps in the flow depends on the other (we first create, then list, and only at the end delete). This is why it has to be done within a single It (the order in a context is not guaranteed)

Reply:
If the calls succeeded then sure, but there is no dependency when every call errors.
And either way I would still suggest they be separate.

Reply:
FWIW I would still like these to be split.
There's no reason for them to be in one test and it should be pretty low effort to separate them

Reply:
I don't understand why this keeps getting resolved.
Am I missing something with how this test works?

It really feels like there's no reason that these should be in the same test case.

Reply:
test case are now separated to individual tests